manifest.php

```diff
@@ -37,7 +37,7 @@
     ),
     'install' => array(
         'php' => array(
-            dirname(__FILE__) . '/scripts/install/createSaveDirectory.php',
+			'oat\taoClientDiagnostic\scripts\install\createDiagnosticTable',
```

Comment:
please use __DIR__ constant to get extension path and DIRECTORY_SEPARATOR instead of \